### DISCUSSION TITLE
Sidebar active state not maintained after form submission in Livewire

### DISCUSSION THREAD

**Discussion by end3r-man** [livewire/livewire#8906] [7. Bugs (archived)]
### Livewire version

3.5.8

### Laravel version

11.9

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Set up a Laravel application with Livewire.
2. Create a sidebar component with menu items that have an active state.
3. In a Livewire component, include the sidebar component and make a form submission.
4. Observe that the active state of the sidebar menu items is lost after the form submission.

### Repository or Wirebox URL

https://github.com/end3r-man/ELOEO

### Additional information

experiencing an issue where the active state of the sidebar menu items is not being maintained after submitting a form.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by omniith** [comment]
put your nav sidebar menu on a different component and load the component via layout. I have yet to find a solution when the component needs livewire property binding updates.
